test-kitchen/skills/omakase-off/references/detailed-workflow.md

```
@@ -153,8 +154,32 @@ For each variant that passed scenarios, use `fresh-eyes`.
 | All fail | Report failures, ask user how to proceed |
 | One survives | Auto-select |
 
-**Step 4: Judge (comparing survivors)**
-Present comparison to user with metrics.
+**Step 4: Invoke Judge Skill**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)